### PR TITLE
Adding Echo for Lemmy to apps list

### PR DESCRIPTION
**Pull request by fishcharlie**
Not 100% sure I did this correctly. So please verify extra closely, and let me know if any adjustments are needed.

### CODE REVIEW

**Comment by Nutomic** [comment]
It needs formatting, try running `npx prettier -w src`

**Comment by fishcharlie** [comment]
@Nutomic Done. Sorry about that.

File: src/shared/components/app-definitions.ts [hunk 1]
```diff
@@ -93,6 +93,22 @@ export const MODERATION_TOOLS: ToolDetails[] = [
...
 
+const ECHO: AppDetails = {
+  name: "Echo",
+  description: "A fully native Lemmy Client for iOS",
+  link: "https://echo.rrainn.com",
+  icon: "/static/assets/images/echo.png",
+  banner: "/static/assets/images/echo_screen.png",
+  links: [
+    {
+      link: "https://echo.rrainn.com/download/iphone",
```
[the remark this answers is not in the document]
Comment (fishcharlie) [reply src/shared/components/app-definitions.ts:104]:
```suggestion
      link: "https://apps.apple.com/us/app/echo-for-lemmy/id6450902296",
```

File: src/assets/images/echo_screen.png
Comment (dessalines) [inline src/assets/images/echo_screen.png:1]: Could you compress / shrink this file? You could use webp like the other files, or some kind of png compression program. Almost all these files are under 100 KB.

**Review by dessalines — CHANGES_REQUESTED** [review]
See comment above.

**Comment by fishcharlie** [comment]
Just made that change.

**Review by dessalines — APPROVED** [review]
Sweet, thx!

**Pull request merged**